doc/sphinx/user_manual/basic_starter_tutorial/16_2D_models.md

```diff
@@ -0,0 +1,73 @@
+(part:user_manual:chap:basic_starter_tutorial:sec:16_2D_models)=
+2D models
+=========
+
+2D models in the world builder are nothing more than a cross section through a 3D model. This means that you need to do to be able to create a 2D model is to state in the world builder file the origin of the cross section (where x=0) and in what direction the cross section should go (the positive y direction). You can set this with the global parameter `cross section`. It takes two points. The first point is the origin, and the second point is the direction of the cross section. 
```

Comment:
```suggestion
2D models in the world builder are nothing more than a cross section through a 3D model. This means that what you need to do to be able to create a 2D model is to state in the world builder file the origin of the cross section (where x=0) and in what direction the cross section should go (the positive y direction). You can set this with the global parameter `cross section`. It takes two points. The first point is the origin, and the second point is the direction of the cross section. 
```

Reply:
I think this could be clearer if either you label the coordinates on your figure to show the cross section coordinates or if you had values in parentheses to your text description. I remember a figure Magali made one time showing the 3D model, and then the cross section line through the 3D model that the 2D model was being made for. This could be helpful for visualizing too what points the cross section is being pulled from. Since you already have the 3D model figures made I imagine this might be a fairly quick edit of just adding a line across the 3D model domain showing where this 2D profile comes from and adding that figure to this page? I am confused here how the first point is supposed to be the origin, but it is listed as 0, 200e3 in the file.

Reply:
I agree, I don't know if I have time to do that before the release though.